### PR TITLE
Default max zoom 22

### PR DESCRIPTION
**Pull request by asheemmamoowala**
Update iOS documentation and use DEFAULT_MAX_ZOOM constant. 
+CHANGELOGs

### CODE REVIEW

File: platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java [hunk 1]
```diff
@@ -508,7 +508,9 @@ public double getMinZoomLevel() {
    * <p>
    * Sets the maximum zoom level the map can be displayed at.
    * </p>
-   *
+   * <p>
+   *   The default maximum zoomn level is 22. The upper bound for this value is 25.5.
```
Comment (1ec5) [inline platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java]: Typo: “zoomn”.

File: platform/ios/src/MGLMapView.h [hunk 2]
```diff
@@ -597,7 +597,8 @@ MGL_EXPORT IB_DESIGNABLE
  * If the value of this property is smaller than that of the
  * minimumZoomLevel property, the behavior is undefined.
  *
- * The default maximumZoomLevel is 20.
+ * The default maximumZoomLevel is 22. The upper bound for this property
```
Comment (1ec5) [inline platform/ios/src/MGLMapView.h]: Property names should be in backticks to benefit from jazzy’s autolinking feature. However, let’s say “default value of this property” here for consistency.

**Comment by friedbunny** [comment]
Are there any prerequisites or blockers that would keep these maxzoom changes from being cherry-picked into the current release branch?

**Comment by asheemmamoowala** [comment]
@friedbunny - I think #9820 and #4168 (for Android only) should be looked at to make sure they are  either acceptable or not an issue.

File: platform/ios/src/MGLMapView.h (hunk 2 above)
Comment (asheemmamoowala) [inline platform/ios/src/MGLMapView.h]: @1ec5 - In this case its referring to itself, so this shouldn't need any linking.

Comment (1ec5) [inline platform/ios/src/MGLMapView.h]: Yeah, that’s why I suggested saying “default value of this property”, same as what you had written for macOS. No big deal though.

**Pull request merged**